docs/docs/packaging/profiles.md

```diff
@@ -166,6 +169,18 @@ port: <int>
 path: <string>
 ```
 
+### `<api>`
+
+Defines the AVS API endpoint that the Node exposes following the [AVS API specification](/docs/metrics/metrics-api).
+
+```yaml
+# Name of the docker-compose service exposing the API
+service: <string>
```

Comment:
I dont see why the name of the service matters. All we need is the port so we can do the health check from it. The port should be specified as "Expose" in the docker-compose file by the AVS developer

Reply:
We need the service name to get the container's IP represented by that service name in the docker-compose file. With the service name, we can get the container ID; with the container ID, we can inspect the container and get the network and IP address. The port might be exposed or not to localhost. This way, we can always access the API.

Reply:
got it, my thought was the port would be exposed. but if this is a usecase we want to support it makes sense